modules/extensions-mule-support/src/main/java/org/mule/runtime/module/extension/mule/internal/loader/parser/MuleSdkOptionalParameterModelParserSdk.java

```diff
@@ -22,10 +22,10 @@ public class MuleSdkOptionalParameterModelParserSdk extends MuleSdkParameterMode
   public MuleSdkOptionalParameterModelParserSdk(ComponentAst parameter, TypeLoader typeLoader) {
     super(parameter, typeLoader);
 
-    parseStructure();
+    parseOptionalStructure();
```

Comment:
bad rename. Not necessarily optional.

Reply:
What I want to do is to avoid the name collision between this method and the parent's one, in order to avoid shadowing and some future confusion (for the moment it's obvious, but who knows...)

It's parsing the part of the structure that it's specific for optional parameters (the default value), so I would rename it to parseOptionalSpecificStructure(), do you have a better name?